ceph/cephfs/cephfs-provisioner.go

```diff
@@ -40,6 +40,7 @@ const (
 	provisionCmd     = "/usr/local/bin/cephfs_provisioner"
 	provisionerIDAnn = "cephFSProvisionerIdentity"
 	cephShareAnn     = "cephShare"
+	defaultNamespace = v1.NamespaceAll
```

Comment:
Instead of setting 'defaultNamespace' to "v1.NamespaceAll" , Isnt it good to set "default"  namespace as the defaultNamespace ?